ensemble-view/src/main/java/hec/ensembleview/ParameterFilter.java

```diff
@@ -0,0 +1,27 @@
+package hec.ensembleview;
+
+import hec.RecordIdentifier;
+
+public class ParameterFilter {
+    public static RecordIdentifier checkParameter(RecordIdentifier recordIdentifier) {
+        String parameter = recordIdentifier.parameter;
+        switch (parameter.toUpperCase()) {
+            case "FLOW":
+            case "MOISTURE DEFICIT":
+            case "SWE":
+            case "PRECIP-CUM":
```

Comment:
Should cumulative precip be removed from this switch statement?

Reply:
agree, removed PRECIP-CUM from the switch statements